common/CMakeLists.txt

```diff
@@ -218,6 +218,17 @@ if(APPLE)
 	target_link_options(common PRIVATE -fobjc-link-runtime)
 endif()
 
+if(USE_DBUS)
+	target_compile_definitions(common PRIVATE USE_DBUS)
+	if(UNIX AND NOT ANDROID AND NOT APPLE)
```

Comment:
There will never be PCSX2 for Android ;)

This should be defined in BuildParameters as an option, inside an `if(LINUX)` check.

Reply:
It's just because freebsd can use dbus.

Reply:
> This should be defined in BuildParameters as an option, inside an if(LINUX) check.
Yes if this is to become the standard way of handling it.

Reply:
Done :+1: